### ISSUE TITLE
Error :- anic: invalid character '<' looking for beginning of value 

### ISSUE THREAD

**Issue by KRiteshchowdary** [alpkeskin/mosint#20]
Getting the following error while trying to run

$ mosint % go run main.go -e example@gmail.com -all
                              _           _
    _ __ ___     ___    ___  (_)  _ __   | |_
   | '_ ` _ \   / _ \  / __| | | | '_ \  | __|
   | | | | | | | (_) | \__ \ | | | | | | | |_
   |_| |_| |_|  \___/  |___/ |_| |_| |_|  \__|
  
   v2.1
   https://github.com/alpkeskin/
   Now: Thursday, 19 May 2022

mosinting  36% |████████████████████████████████████████████████                                                                                        | (36/100, 6 it/s) [21s:10s]panic: invalid character '<' looking for beginning of value

goroutine 1 [running]:
main/modules.doRequest({0xc000d3e0f0, 0x4e}, {0x16674ff, 0x7})
	~/mosint/modules/ThreatCrowd.go:28 +0x670
main/modules.RelatedDomains(...)
	~/mosint/modules/ThreatCrowd.go:52
main.main()
	~/mosint/main.go:263 +0xc2b
exit status 2

<img width="1275" alt="Screenshot 2022-05-19 at 11 55 30 AM" src="https://user-images.githubusercontent.com/44915939/169225061-23b5d423-e958-42f2-946f-07cff8a46b17.png">

**Comment by gray-area** [comment]
I am getting the same issue on Kali.

**Comment by gray-area** [comment]
I retested this, and you are right. However, I was able to re-create the fix. You actually need to run the command without sudo, it will fail. Then run the command as sudo and it will run. Once or twice, it took me 3 times of running the command for it to run without error. I am not sure what the fix is, but this is a workaround to getting it to run. I will try and dig into the issue and see if I can help find the fix too.
